### PR TITLE
Adding RWL021 Philips Hue Dimmable Switch compatibility

### PR DESCRIPTION
**Pull request by Armageddon95**
I can't test the code yet. Please try it.

### CODE REVIEW

File: src/accessories/philips/philips-hue-dimmable-switch.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        this.switchServiceBrightUp
+          .getCharacteristic(ProgrammableSwitchEvent)
+          .setValue(ProgrammableSwitchEvent.LONG_PRESS);
+        break;
+      case 'brightness_down_hold':
+        this.switchServiceBrightDown
+          .getCharacteristic(ProgrammableSwitchEvent)
+          .setValue(ProgrammableSwitchEvent.LONG_PRESS);
+        break;
+
```
Comment (madchicken) [inline src/accessories/philips/philips-hue-dimmable-switch.ts:93]: I don't think this switch/case is right. Reading the description page [here](https://www.zigbee2mqtt.io/devices/324131092621.html#action-enum) I see different actions exposed by this device

File: src/index.ts [hunk 2]
```diff
@@ -68,6 +69,11 @@ function registerSupportedDevices(): void {
     ],
     PhilipsHueWhiteAndColor
   );
+  registerAccessoryClass(
+    'Philips',
+    ['RWL021'],
```
Comment (madchicken) [inline src/index.ts:74]: I think you should use `324131092621` as model here

**Review by madchicken — CHANGES_REQUESTED** [review]

File: src/accessories/philips/philips-hue-dimmable-switch.ts (hunk 1 above)
Comment (tr1ng0) [reply src/accessories/philips/philips-hue-dimmable-switch.ts:93]: @Armageddon95 do you need help with the pull request? Madchicken is correct with those request changes

**Comment by madchicken** [comment]
Closing stale PR

**Pull request closed without merging**